src/Numerics/Optimizers/ExhaustiveOptimizer/Documentation.rst

```diff
@@ -27,6 +27,15 @@ C++
 .. literalinclude:: Code.cxx
    :lines: 18-
 
+
+Jupyter Notebook
+-----------------
+
+.. image:: https://mybinder.org/badge_logo.svg
+ :target: https://mybinder.org/v2/gh/InsightSoftwareConsortium/ITKExamples/master
```

Comment:
This should point directly to the notebook:

https://mybinder.org/v2/gh/InsightSoftwareConsortium/ITKExamples/master?filepath=src%2FNumerics%2FOptimizers%2FExhaustiveOptimizer%2FPlotExhaustiveOptimizer.ipynb

Reply:
Fixed. Note that this link will not work until the merge completes and the notebook file exists in `master`. The equivalent link (while this branch exists only) is [https://mybinder.org/v2/gh/tbirdso/ITKExamples/exhaustive-notebook?filepath=src%2FNumerics%2FOptimizers%2FExhaustiveOptimizer%2FPlotExhaustiveOptimizer.ipynb](https://mybinder.org/v2/gh/tbirdso/ITKExamples/exhaustive-notebook?filepath=src%2FNumerics%2FOptimizers%2FExhaustiveOptimizer%2FPlotExhaustiveOptimizer.ipynb) , which I have verified launches and runs the notebook successfully.